README.md

```diff
@@ -827,12 +827,12 @@ Returns: `string`
 
 Usage:
 ```
-{{currency 1000000 code='USD'}}  => $1,000,000.00
-{{currency 1000000 code='EUR'}}  => 1 000 000,00 €
-{{currency 1000000 code='EUR' precision=0}}  => 1 000 000 €
+{{formatCurrency 1234567.89 currency='USD'}}  => $1,234,567.89
```

Comment:
`formatCurrency` is fine as it aligns with `formatDate` but we need to make `currency='USD'` to `code='USD'` just like what @kabirbaidhya did there.